.github/workflows/release.yml

```diff
@@ -64,3 +64,30 @@ jobs:
       - name: Publish
         run: |
           make -f builder/Makefile.release publish SNAPSHOT_VERSION=${{ github.ref_name }}
+  build-docker-image:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up QEMU
```

Comment:
Why is it necessary to install QEMU?

Reply:
> docker/setup-qemu-action configures QEMU emulation for the Arm portion of our multi-platform Docker image build. This is required for multi-platform Docker image builds in GitHub Actions, as the hosted runners are Intel machines.


ref:https://docs.docker.com/build/ci/github-actions/multi-platform/